lib/pages/contacts_tab/contacts_appbar_style.dart

```diff
@@ -1,24 +1,40 @@
+import 'package:fluffychat/di/global/get_it_initializer.dart';
+import 'package:fluffychat/utils/responsive/responsive_utils.dart';
 import 'package:flutter/cupertino.dart';
 
 class ContactsAppbarStyle {
+  static ResponsiveUtils responsiveUtils = getIt.get<ResponsiveUtils>();
   static const Size preferredSizeAppBar = Size.fromHeight(120);
-
   static const double iconSize = 24.0;
 
-  static const double textFieldHeight = 64.0;
+  static const double textFieldHeight = 56.0;
 
   static const double toolbarHeight = 56.0;
 
   static const double leadingWidth = 76.0;
 
   static const double textFieldBorderRadius = 24.0;
 
-  static AlignmentGeometry alignment = AlignmentDirectional.centerStart;
+  static AlignmentGeometry alignmentTitle(context) =>
+      ResponsiveUtils().isMobile(context)
+          ? Alignment.center
+          : AlignmentDirectional.centerStart;
 
   static EdgeInsetsDirectional appbarPadding =
       const EdgeInsetsDirectional.symmetric(
     horizontal: 16.0,
   );
 
   static EdgeInsetsDirectional contentPadding = EdgeInsetsDirectional.zero;
+  static const double textStyleHeight = 24 / 17;
```

Comment:
what is it? can you explain it?

Reply:
The default height of the text( 17 is this case) will be multiplied by this, to make the height of the text 24
![image](https://github.com/user-attachments/assets/80dcfcb3-354d-41f8-bdea-902ff80765ef)